tests/__tests__/get-cache-key.spec.ts

```diff
@@ -1,47 +1,34 @@
-import { getCacheKey } from '../../src/preprocessor';
-import { TransformOptions } from '../../src/jest-types';
+import getCacheKey from '../../dist/utils/get-cache-key';
+import cfg from '../__helpers__/jest-config';
 
 describe('getCacheKey', () => {
   const src = 'console.log(123);';
-  const filepath = '/tmp/filepath';
-  const configStr = `{
-    "globals": {
-      "__TS_CONFIG": {
-        "compilerOptions": {
-          "target": "ES5",
-          "module": "commonjs"
-        }
-      }
-    },
-    "transform": {
-      "^.+\\\\.tsx?$": "../../preprocessor.js"
-    },
-    "testRegex": "(/__tests__/.*|(\\\\.|/)(test|spec))\\\\.(jsx?|tsx?)$"
-  }`;
-  const options: TransformOptions = { instrument: false };
+  const jestConfig = cfg.simple(null, {
+    transform: { '^.+\\\\.tsx?$': '../../preprocessor.js' },
+    testRegex: '(/__tests__/.*|(\\\\.|/)(test|spec))\\\\.(jsx?|tsx?)$',
+  });
+  const filepath = `${jestConfig.rootDir}/some-file.ts`;
+  const configStr = JSON.stringify(jestConfig);
+  const options = { instrument: false, rootDir: jestConfig.rootDir };
   const originalHash = getCacheKey(src, filepath, configStr, options);
 
   it('should change hash when src changes', () => {
     const newSrc = 'console.log(1234);';
-    const newHash = getCacheKey(newSrc, filepath, configStr);
+    const newHash = getCacheKey(newSrc, filepath, configStr, options);
     expect(newHash).not.toBe(originalHash);
   });
 
   it('should change hash when filepath changes', () => {
-    const newPath = '/tmp/newfilepath';
-    const newHash = getCacheKey(src, newPath, configStr);
-    expect(newHash).not.toBe(originalHash);
-  });
-
-  it('should change hash when tsconfig changes', () => {
```

Comment:
why was this removed?

Reply:
Hmmm... I think I forgot something here yeah. Will add a test for when tsconfig changes.